python/CommonFramework.cpp

```diff
@@ -57,13 +57,22 @@ bool PythonBase::Initialize()
         // provided by the system). These API calls have been added in an attempt to
         // solve the problems. Not sure if they are really required, but better save
         // than sorry... ;)
-        strcpy(m_home_dir, GetPythonHome().string().c_str());
```

Comment:
Everything around wchar_t in an unspecified mess, stay utf-8 and convert when calling the platform specific function.

Reply:
From Python 3 API:

> Use Py_DecodeLocale() to decode a bytes string to get a wchar_* string.

https://docs.python.org/3/c-api/init.html#c.Py_SetPythonHome

Reply:
The only platform specific functions are `Py_SetProgramName` and `Py_SetPythonHome` and `wchat_t` array aren't used somewhere else.
Should I use `Py_DecodeLocale` to get `wchar_t*` instead of `L` prefix?

Also I have to stay with `wchar_t` because:

> The argument should point to a zero-terminated wide character string in static storage whose contents will not change for the duration of the program’s execution. No code in the Python interpreter will change the contents of this storage.

Reply:
Changed to use `Py_DecodeLocale`.